WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -466,12 +462,6 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
     }
 
     private func handleTrashPage(_ post: AbstractPost) {
```

Comment:
We can remove this entire method. Apologies for not doing this in the context menu PR!